### PR TITLE
Add animatron.local.js bundle without analytics

### PR DESCRIPTION
**Pull request by RobertoUa**
@shamansir pls review

### CODE REVIEW

File: src/anm/utils.js [hunk 1]
```diff
@@ -277,6 +277,13 @@ function makeApiUrl(prefix, path, loadSrc) {
     }
 }
 
+function getObjectId () {
```
Comment (shamansir) [inline src/anm/utils.js]: what's the major difference with `utils.guid`, more reliable random? :)

**Comment by shamansir** [comment]
Looks great in general, nice! Thank you!

File: src/anm/utils.js (hunk 1 above)
Comment (RobertoUa) [inline src/anm/utils.js]: It's not about random, it's about ObjectId.

Comment (shamansir) [inline src/anm/utils.js]: You mean it's compliant to a spec? Ok, got it.

**Pull request merged**